.netlify/functions-internal/chunks/vnode-children_3769332a.mjs

```
@@ -1,37 +1,47 @@
 let ids = 0;
 function convert(children) {
-	const nodeMap = new WeakMap();
-	let doc = parse(children.toString().trim());
-	let id = ids++;
-	let key = 0;
-	let root = createElement(Fragment, { children: [] });
+  const nodeMap = new WeakMap();
+  let doc = parse(children.toString().trim());
+  let id = ids++;
+  let key = 0;
+  let root = createElement(Fragment, { children: [] });
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use additional arguments to React.createElement

(lint/correctness/noChildrenProp)